jujugui/static/gui/src/app/app.js

```diff
@@ -3283,44 +3281,28 @@ YUI.add('juju-gui', function(Y) {
     'net-utils',
     // React components
     'account',
-    'added-services-list',
```

Comment:
As components are converted these are removed from app.js and the YUI instance. Eventually this will only have the Models/Topology, etc